### PR TITLE
[#5379] Add basic Fileset commands to Gravitno CLI

### PR DESCRIPTION
**Pull request by justinmclean**
### What changes were proposed in this pull request?

Added basic Fileset commands to Gravitno CLI

### Why are the changes needed?

To expand Gravitino CLI support.

Fix: #5379

### Does this PR introduce _any_ user-facing change?

No, but it adds extra commands to the Gravitino CLI.

### How was this patch tested?

Compiled and tested locally.

### CODE REVIEW

**Comment by justinmclean** [comment]
@shaofengshi once the owner PR is merged this is ready for review or you could just review the fileset parts

**Comment by justinmclean** [comment]
@shaofengshi @jerryshao this is ready for review

File: docs/cli.md [hunk 1]
```diff
@@ -591,3 +596,29 @@ gcli group grant --group groupA --role admin
...
+#### Create a fileset
+
+```bash
+gcli fileset create --name hadoop.fileset.schema --fileset example --properties managed=true,location=file:/tmp/root/schema/example
+```
+
+#### List filesets
+
+```bash
+gcli fileset list --name hadoop.fileset.schema
```
Comment (shaofengshi) [inline docs/cli.md:611]: what's the "hadoop" mean? is it the metalake name or catalog name? Besides, this should be in the same style with table's command.

Comment (justinmclean) [reply docs/cli.md:611]: it's a catalog name

Comment (shaofengshi) [reply docs/cli.md:611]: if "hadoop" is catalog, then "fileset" is the schema name, what the "schema" represents?

**Review by shaofengshi — APPROVED** [review]
LGTM; Justin, thank you for the changes.

**Pull request merged**